x-pack/plugins/actions/server/lib/action_executor.ts

```diff
@@ -218,6 +218,7 @@ export class ActionExecutor {
             },
             { configurationUtilities }
           );
+          const loggerId = actionTypeId.startsWith('.') ? actionTypeId.substring(1) : actionTypeId;
```

Comment:
Similar comment here as alerting task runner. I would initialize the new logger with the actionTypeId on line 146 when you get the `actionTypeId` from the `actionInfo` :)

Reply:
Resolved in https://github.com/elastic/kibana/pull/142121/commits/934714706143231b309afe0d23c935924441488c